src/main/java/org/jitsi/stats/media/AbstractStatsPeriodicRunnable.java

```diff
@@ -107,40 +107,29 @@ public AbstractStatsPeriodicRunnable(
             }
         }
 
-        try
-        {
-            EntityBareJid roomJid = JidCreate.entityBareFrom(conferenceJid);
-
-            String conferenceName = roomJid.getLocalpart().toString();
+        String conferenceName = conferenceJid.getLocalpart().toString();
 
-            // extract siteId/subdomain
-            String domain = roomJid.getDomain().toString();
-            if (conferenceIDPrefix != null
-                && domain.endsWith("." + conferenceIDPrefix))
+        // extract siteId/subdomain
```

Comment:
Can you add an example to the docs?

If I understand correctly if the prefix is set to `configured.domain`, then the JID `name@anything.site.id.configured.domain` will be translated to `configured.domain/site.id/name`.

If the domain is not set, it will be just `name`.

And if the prefix is configured, but the JID doesn't match it will be `configured.domain/name`.



The `conferenceName` variable is only used in the end, so the logic can be simplified to:

```java
        if (conferenceIDPrefix != null)
        {
            conferenceIDBuilder.append(conferenceIDPrefix);
            if (!conferenceIDPrefix.endsWith("/"))
            {
                conferenceIDBuilder.append("/");
            }

            // extract siteId/subdomain
            String domain = conferenceJid.getDomain().toString();
            if (domain.endsWith("." + conferenceIDPrefix))
            {
                // strip `.conferenceIDPrefix`
                String mucDomain = domain.substring(0, domain.indexOf(conferenceIDPrefix) - 1);
                int dotIx = mucDomain.indexOf('.');
                if (dotIx > 0)
                {
                    String siteId = mucDomain.substring(dotIx + 1);

                    if (!StringUtils.isNullOrEmpty(siteId))
                    {
                        this.initiatorSiteID = siteId;
                        conferenceIDBuilder.append(siteId).append("/");
                    }
                }
            }
        }

        conferenceIDBuilder.append(conferenceJid.getLocalpart().toString());
```